react/src/components/AssetsPanel/AssetsPanel.tsx

```diff
@@ -102,6 +111,7 @@ const AssetsPanel: React.FC<Props> = ({
         <FileBrowserModal
           isOpen={isModalOpen}
           toggle={toggleModal}
+          onImported={handleFileImport}
```

Comment:
Just noting this - for long asset lists, the panel doesn't scroll correctly and you can't see the download JSON button. Not an issue with this PR but to document for testing tasks.
![Screenshot 2025-02-06 at 3 11 28 PM](https://github.com/user-attachments/assets/cb074310-0244-40ef-8d88-cb909d32f87e)

Reply:
Thanks. regression outside this PR? 😬    putting in a new jira issue to track it:  https://tacc-main.atlassian.net/browse/WG-425